File: src/sdk-reference/php/3/security/replace-user/snippets/replace-user-1.php

<?php

use \Kuzzle\Kuzzle;
use \Kuzzle\Security\User;

$kuid = 'myUser';
$userDefinition = [
  'profileIds' => ['admin'],
  'firstname' => 'My Name Is',
  'lastname' => 'Jonas'
];

$kuzzle = new Kuzzle('localhost');
$security = $kuzzle->security();

try {
  $user = $security->replaceUser($kuid, $userDefinition);

  // $user instanceof User
}
catch (ErrorException $e) {

}

File: src/sdk-reference/android/3/security/replace-user/snippets/replace-user-1.java

JSONObject newContent = new JSONObject()
  .put("profileIds", new JSONArray()
    .put("admin")
  )
  .put("firstname", "My Name Is")
  .put("lastname", "Jonas");

kuzzle
  .security
  .replaceUser("User ID", newContent, new ResponseListener<User>() {
    @Override
    public void onSuccess(User user) {

    }

    @Override
    public void onError(JSONObject error) {

    }
  });
